app/util.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const request = require('request-promise');
+
+// returns a Date representing tomorrow at 12pm so that
+// we can issue darksky 'time machine' requests to get an
+// idea of the weather tomorrow at a time that should be
+// relevant to someone interested in the weather
+function getTomorrowAtNoon() {
+    let tomorrow = new Date();
+    tomorrow.setDate(tomorrow.getDate() + 1);
+    tomorrow.setHours(12);
+    return tomorrow;
+}
+
+// fixed coordinates used to query darksky
+const latitude = 38.8892681;
+const longitude = -77.0501425;
+const darkskyToken = process.env.DARKSKY_TOKEN || '';
+const darkskyUrlBase = `https://api.darksky.net/forecast/${darkskyToken}/${latitude},${longitude}`;
+
+// returns a promise that resolves to the JSON representing the current weather
+// report from darksky on success
+function getCurrentWeatherReport() {
+    // we don't need the minutely or hourly forecast, so we'll exclude them
+    const darkskyCurrentlyUrl = `${darkskyUrlBase}` + '?exclude=minutely,hourly';
+    return request({ uri: darkskyCurrentlyUrl, json: true });
+}
+
+function getTomorrowsWeatherReport() {
+    const darkskyTomorrowUrl = `${darkskyUrlBase}` + getTomorrowAtNoon().getTime() + '?exclude=currently,minutely,hourly';
+    return request({ uri: darkskyTomorrowUrl, json: true });
+}
+
+function generateCurrentSummary(dataBlock) {
+    return generateWeatherReport(dataBlock, 'is currently', 'feels like');
+}
+
+function generateTomorrowsSummary(dataBlock) {
+    return generateWeatherReport(dataBlock, 'tomorrow will be', 'will feel like');
+}
+
+// maps the darksky 'icon' field to an appropriate emoji representation!
+// not supported by jshint yet - https://github.com/jshint/jshint/pull/2413
+const icon2Emoji = {
```

Comment:
Yeah, that should work. I was concerned about having to reverse engineer the emoji into its actual unicode code point since the display of emojis with certain fonts can sometimes make it hard to see which one is actually being used. 

Figured if someone wanted to change them, it would be easier to look up the code point, rather than searching for something like 'smiling moon face emoji':

🌝 <-- \:full_moon_with_face
🙂 <-- \:slightly_smiling_face